pdns/responsestats.cc

```diff
@@ -8,7 +8,7 @@
 
 #include "dnsparser.hh"
 
-ResponseStats::ResponseStats() :   d_qtypecounters(new std::atomic<unsigned long>[65536])
+ResponseStats::ResponseStats() :   d_qtypecounters(new std::atomic<unsigned long>[65536]), d_rcodecounters(new std::atomic<unsigned long>[65536])
```

Comment:
We don't need 65536 entries for the response code.
```suggestion
ResponseStats::ResponseStats() :   d_qtypecounters(new std::atomic<unsigned long>[65536]), d_rcodecounters(new std::atomic<unsigned long>[256])
```

Reply:
Thanks, I didnt noticed that 65536 was because it's the uint16_t max value. rcode is uint8_t :)
I just updated this file